src/osd/osd_types.h

```diff
@@ -4412,15 +4412,42 @@ static inline ostream& operator<<(ostream& out, const notify_info_t& n) {
 	     << " " << n.timeout << "s)";
 }
 
+struct chunk_info_t {
+  enum {
+    CHUNK_CLEAN = 0,
+    CHUNK_DIRTY = 1, 
```

Comment:
if these are flags, we just need FLAG_DIRTY = 1, and the lack of that flag means clean